core/action.py

```diff
@@ -67,6 +67,9 @@ def process(self, submission, last_good_submission=None):
         if submission_is_meta in self.issues:
             message_lines.append("https://www.reddit.com{} is a meta post switcharoo".format(
                 submission.permalink))
+        if submission_is_NSFW in self.issues:
```

Comment:
You'll see this with the new issue tracking system, the check should now be `if issues.submission_is_NSFW:`